frontend/src/authentication/hooks/useToken.js

```diff
@@ -21,14 +23,18 @@ export const useToken = () => {
               await auth.signinSilent();
             }
             const t = auth.user.id_token;
+            const at = auth.user.access_token;
```

Comment:
Are we now using both Access and Identity token ?

Reply:
Do we need any particular token for any specific use ? Like a specific gql endpoint needs an identity token and cannot work with access token ?

Reply:
debating whether we should just send access token and forget about ID

currently we send both which may be overkill

ID token contains headers like `email` that we do need. However when we run a request with access token against `userinfo` endpoint we get that information (such as `email`) in response that we can add to context


Additionally, access tokens (for cognito at least) contain user and group information - not sure if that is standard

Reply:
At a high level the token uses are:
-  the access token is needed to invoke the `/userinfo` endpoint
- the verified claims of the ID Token are needed to populate context information (email, user, groups) for downstream lambda (this theoretically could be sourced elsewhere)